src/classes/UTIL_LightningComponentSupport.cls

```diff
@@ -36,23 +36,45 @@
 
 public with sharing class UTIL_LightningComponentSupport {
 
+    public static final String LIGHTNING_DATA_TYPE_NUMBER = 'number';
+    public static final String LIGHTNING_DATA_TYPE_TEXT = 'text';
+    public static final String LIGHTNING_DATA_TYPE_CHECKBOX = 'checkbox';
+    public static final String LIGHTNING_DATA_TYPE_DATE = 'date';
+    public static final String LIGHTNING_DATA_TYPE_DATE_LOCAL = 'date-local';
+
+    /*******************************************************************************************************
+    * @description Converts Apex field data type to type expected by lightning:datatable.
+    * @param rawType: the Schema.DisplayType from DescribeFieldResult
```

Comment:
Ok last change -- can you rename the param to fieldDisplayType? Then we're all good!